default_multi_user/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1,3 @@
+* Lois Rilo <lois.rilo@forgeflow.com>
+* Sunanda Chhatbar <sunanda.chhatbar@initos.com>
+* Dhara Solanki <dhara.solanki@initos.com>
```

Comment:
As fare as I know are migrations without any code change not enough to put you on the contributor list.